mds/migrations/0044_auto_20190906_0818.py

```diff
@@ -0,0 +1,52 @@
+# Generated by Django 2.1.11 on 2019-09-06 08:18
```

Comment:
Rename the automatically generated filename with an explicit name.

Reply:
C'est fait ;)